Spec/Auth.swift

```diff
@@ -713,6 +713,67 @@ class Auth : QuickSpec {
                 }
 
             }
+
+            // RSA8d
+            context("When authCallback option is set, it will invoke the callback") {
+
+                pending("with a token string") {
+                    let options = AblyTests.clientOptions()
+                    let expectedTokenParams = ARTTokenParams()
+
+                    options.authCallback = { tokenParams, completion in
+                        expect(tokenParams).to(beIdenticalTo(expectedTokenParams))
+                        // TODO: call the completion with a token string
+                        //completion("token_string", nil)
```

Comment:
Because it doesn't compile. The completion only receives a `TokenDetails` but I saw on other libs that it is also possible to call it with a Token string.